common/src/main/java/com/wynntils/handlers/chat/ChatHandler.java

```diff
@@ -118,6 +118,18 @@ public void onChatReceived(ChatPacketReceivedEvent e) {
             }
             // If we are not separating NPCs, just pass the chat screen right on. We can do
             // nothing about it.
+            // FIXME: This is a temporary solution. We don't classify NPC messages correctly, they are marked as INFO.
+            //        Even if we don't separate NPC messages, we should still classify them correctly.
+            //        However, this hack allows us to disable the NPC dialog extraction feature (/overlay), and still
+            // get
+            //        the dialog in the chat window.
```

Comment:
Thank you spotless for producing this utter garbage.

Reply:
I thought Spotless stayed out of comments?

Reply:
No, line width applies